src/apps/vpn/ui/screens/getHelp/developerMenu/ViewDeveloperMenu.qml

```diff
@@ -69,13 +69,6 @@ MZViewBase {
             Component.onCompleted: {
                 serverAddressInput.text = MZSettings.stagingServerAddress;
             }
-
-            Component.onDestruction: {
-                // If a server address is not specified, don't use a staging server
-                if (serverAddressInput.text.length == 0) {
-                    MZSettings.stagingServer = false;
-                }
-            }
```

Comment:
Any reason not to keep the destruction code. It's a rare use case (since changing the server requires an app restart), but I don't see any harm in resetting it on destruction.

The only question I have is, if we destruct while `stagingServer === true` and `serverAddressInput.text === 0`, should we set them to production, or keep them on staging and just re-add default address

Reply:
The destruction code runs when the view changes, but not when the window closes. (This may be a Qt problem .) Given, this inconsistency I think it may be better to handle it on app restart in AppConstants::setStaging().

Reply:
Same as above, since we have to restart the app anyways, no need to really do anything on destruction.

The only slightly confusing thing here is that if the URL in the textfield is blank, and staging is turned on, theres really no way for the user to know that we will revert back to the default staging URL on restart, but I don't think thats a huge deal.